src/state/selectors/searches.js

```diff
@@ -83,3 +85,25 @@ export const getSelectedContentSearchAnnotations = createSelector(
     ),
   })).filter(val => val.resources.length > 0),
 );
+
+export const getResourceAnnotationForSearchHit = createSelector(
+  [
+    getSearchAnnotationForCompanionWindow,
+    (state, { hitId }) => hitId,
```

Comment:
Is `hitId` the right name? Isn't this an annotation uri?

Reply:
It's the it of the annotation in the hit (from https://github.com/ProjectMirador/mirador/blob/0a375b6e4bf5bc4c91976785fd7337ab21f88a05/src/containers/SearchHit.js#L26).  My intention was to differentiate it from the id in the resource, but agree that `annotationUri` may be less ambiguous.

Reply:
`hitAnnotationUriThisIsTheOne`? :trollface: